### PR TITLE
Kelvin's Joke

### PR DESCRIPTION
**Pull request by cteas**
This addressees issue #1. Adds a dad joke.

### CODE REVIEW

File: DadJokes/JokeStore.swift [hunk 1]
```diff
@@ -10,6 +10,7 @@ struct JokeStore {
   
   /// we NEED more jokes here!!!
   private var storage: [Joke] = [
+    Joke(description: "You should be careful with functions that return a value...", punchline: "In fact, I would just a void them."),
```
Comment (cteas) [inline DadJokes/JokeStore.swift:13]: LOL, my sides :+1:

**Review by cteas — COMMENTED** [review]
So funny!

File: DadJokes/JokeStore.swift (hunk 1 above)
Comment (darren102) [inline DadJokes/JokeStore.swift:13]: Good One

**Review by darren102 — APPROVED** [review]
Very funny, side splitting

**Pull request closed without merging**